R/utils-generators.R

```diff
@@ -47,10 +48,10 @@ gen_ages <- function(dis_output, numcases, set_age_na = TRUE) {
 
   if (has_value(age_month_var) && length(U2_YEARS) > 0 && sum(U2_YEARS) > 0) {
     # age_month
-    months <- sample_age(24L, length(U2_YEARS))
+    months <- sample_age(year_cutoff + 1 * 12, length(U2_YEARS))
```

Comment:
Remember that R respects order of operations
```suggestion
    months <- sample_age((year_cutoff + 1) * 12, length(U2_YEARS))
```